client/pages/Dashboard/index.jsx

```diff
@@ -0,0 +1,142 @@
+// React lib import
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+
+// third-party imports
+import { Route, Redirect, Switch } from 'react-router-dom';
+
+// components
+import Header from '../../components/common/Header';
+import { HomePage } from '../../components/common/HomePage';
+import EventPage from '../../components/Events/index';
+
+// stylesheet
+import '../../assets/style.scss';
+
+// thunk
+import {
+  loadActiveUser,
+  displayLoginErrorMessage,
+} from '../../actions/userActions';
+
+// utils
+import isLoggedIn from '../../utils/isLoggedIn';
+import isTokenExpired from '../../utils/isTokenExpired';
+
+/**
+ * Represents Dashboard component
+ *
+ * @class Dashboard
+ * @extends {React.Component}
+ */
+class Dashboard extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      activeUser: {},
+    };
+  }
+
+  /**
+   * React Lifecycle hook
+   *
+   * @memberof Dashboard
+   * @returns {null}
+   */
+  componentDidMount() {
+    this.props.loadActiveUser();
+  }
+
+
+  /**
+   * React Lifecycle hook
+   *
+   * @param {nextProps, prevState}
+   * @memberof Dashboard
+   * @returns {null}
+   */
+  static getDerivedStateFromProps(nextProps, prevState) {
+    if (nextProps.activeUser
+      && nextProps.activeUser === prevState.activeUser) {
+      return null;
```

Comment:
Shouldn't be returning null also. Since you don't want anything to happen if this conditions are met, why not put the condition in which you want something to happen first.